internal/i18n/en_translations_descriptions.go

```diff
@@ -0,0 +1,544 @@
+	SubscriptionCoreOptionsFirstEvent = ffm("SubscriptionCoreOptions.firstEvent", "Whether your appplication would like to receive events from the 'earliest' event emitted by your FireFly node (from the beginning of time), the 'latest' event (from now), or a specific event sequence. Default is 'latest'")
+	SubscriptionCoreOptionsReadAhead  = ffm("SubscriptionCoreOptions.readAhead", "The number of events to stream ahead to your application, while waiting for confirmation of consumption of those events. At least once delivery semantics are used in FireFly, so if your application crashes/reconnects this is the maximum number of events you would expect to be redelivered after it restarts")
+	SubscriptionCoreOptionsWithData   = ffm("SubscriptionCoreOptions.withData", "Whether message events delivered over the subscription, should be packaged with the full data of those messages in-line as part of the event JSON payload. Or if the application should make separate REST calls to download that data")
+
+	// TokenApproval field descriptions
+	TokenApprovalLocalID         = ffm("TokenApproval.localId", "The UUID of this token approval, in the local FireFly node")
+	TokenApprovalPool            = ffm("TokenApproval.pool", "The UUID the token pool this approval applies to")
+	TokenApprovalTokenIndex      = ffm("TokenApproval.tokenIndex", "The index of the token within the pool that this approval applies to")
+	TokenApprovalConnector       = ffm("TokenApproval.connector", "The token connector. Required on input when there are more than one token connectors configured")
+	TokenApprovalKey             = ffm("TokenApproval.key", "The blockchain signing key for the approval request. On input defaults to the first signing key of the organization that operates the node")
+	TokenApprovalOperator        = ffm("TokenApproval.operator", "The blockchain identity that is granted the approval")
+	TokenApprovalApproved        = ffm("TokenApproval.approved", "Whether this record grants permission for an operator to perform actions on the token balance (true), or revokes permission (false)")
+	TokenApprovalInfo            = ffm("TokenApproval.info", "Token connector specific information about the approval operation, such as whether it applied to a limited balance of a fungible token. See your chosen token connector documentation for details")
+	TokenApprovalNamespace       = ffm("TokenApproval.namespace", "The namespace for the approval, which must match the namespace of the token pool")
+	TokenApprovalProtocolID      = ffm("TokenApproval.protocolId", "The protocol identifier of the blockchain event for the token approval. See the 'protocolId' field on blockchain events for more information")
+	TokenApprovalCreated         = ffm("TokenApproval.created", "The creation time of the token approval")
+	TokenApprovalTX              = ffm("TokenApproval.tx", "If submitted via FireFly through the local node, this will reference the UUID of the FireFly transaction")
```

Comment:
This isn't specific to the local node... I think just `If submitted via FireFly, this will reference the UUID of the FireFly transaction`

Caveat that the token connector must support attaching data for this to be true... although there are some planned enhancements to that in the near term.